src/Packet.cpp

```diff
@@ -151,6 +151,44 @@ void Packet::copyInto(Packet& pack) {
   strcpy(pack.data, data);
 }
 
+void Packet::setPacketData(char charArray[240]){
+  strcpy(data, charArray);
+}
+
+void Packet::setPacketData(String wordy){
+  char charArray[240];
+  wordy.toCharArray(charArray, sizeof(wordy));
+  this->setPacketData(charArray);
+}
+
+void Packet::setPacketData(float f) {
+  uint8_t _decimals = 3;
+  int i = f;
+  float res = f - i;
+  if (res == 0) {
+    char charArray[240];
+    itoa(i, charArray, 10);
+    this->setPacketData(charArray);
+  } else {
+    String floatString = String(f, _decimals);
+    this->setPacketData(floatString);
+  }
+}
+
+void Packet::setPacketData(float f, uint8_t decimals) {
```

Comment:
This is not a good way to overload functions - you have basically the same code twice here in both implementations of `setPacketData`.

Delete the previous `setPacketData` (line 164) and in this one, add default value to the `decimals` argument in Packet.h